python/cudf/cudf/core/column/column.py

```diff
@@ -363,7 +407,7 @@ def nullmask(self) -> Buffer:
         """The gpu buffer for the null-mask"""
         if not self.nullable:
             raise ValueError("Column has no null mask")
-        return self.mask_array_view
+        return self.mask_array_view(mode="write")
```

Comment:
No, I think it can be `mode="read"`.
But notice, the access mode doesn't affect the spillability anymore. It is only whether `get_ptr(...)` is called within `acquire_spill_lock` or not that determines if the buffer is marked unspillable temporarily or permanently.